### DISCUSSION TITLE
Adding a nonce to the javascript tag on 1.x possible?

### DISCUSSION THREAD

**Discussion by ckovey** [livewire/livewire#3444] [7. Bugs (archived)]
Hello, we are using a CSP rule of `script-src 'nonce-xxx' 'strict-dynamic' https:;` for our Laravel 6.x site which almost works with livewire 1.x

A nonce is added to the inline script, but not the script source.  Changing https://github.com/livewire/livewire/blob/1.x/src/LivewireManager.php to use

    <script src="{$fullAssetPath}" data-turbolinks-eval="false"{$nonce}></script>
    <script data-turbolinks-eval="false"{$nonce}>

Resolves the CSP warning.  Am I missing an issue with adding this nonce to the script src?

master seems to have this change https://github.com/livewire/livewire/blob/master/src/LivewireManager.php#L277

**Comment by joshhanley** [comment]
@ckovey I'm sure a PR to the 1.x branch would be welcome

**Reply by ckovey** [reply]
I'm already using my fork for our app, so I had a PR ready! https://github.com/livewire/livewire/pull/3481

**Reply by joshhanley** [reply]
Nice!
